2025-06-25 11:04:22,154 INFO    MainThread:281514 [wandb_setup.py:_flush():81] Current SDK version is 0.20.1
2025-06-25 11:04:22,154 INFO    MainThread:281514 [wandb_setup.py:_flush():81] Configure stats pid to 281514
2025-06-25 11:04:22,154 INFO    MainThread:281514 [wandb_setup.py:_flush():81] Loading settings from /home/thiago-bittencourt/.config/wandb/settings
2025-06-25 11:04:22,154 INFO    MainThread:281514 [wandb_setup.py:_flush():81] Loading settings from /home/thiago-bittencourt/fc-projeto-final/wandb/settings
2025-06-25 11:04:22,154 INFO    MainThread:281514 [wandb_setup.py:_flush():81] Loading settings from environment variables
2025-06-25 11:04:22,154 INFO    MainThread:281514 [wandb_init.py:setup_run_log_directory():703] Logging user logs to /home/thiago-bittencourt/fc-projeto-final/wandb/run-20250625_110422-u1yysjyp/logs/debug.log
2025-06-25 11:04:22,154 INFO    MainThread:281514 [wandb_init.py:setup_run_log_directory():704] Logging internal logs to /home/thiago-bittencourt/fc-projeto-final/wandb/run-20250625_110422-u1yysjyp/logs/debug-internal.log
2025-06-25 11:04:22,155 INFO    MainThread:281514 [wandb_init.py:monkeypatch_ipython():622] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7bdb2e76d250>
2025-06-25 11:04:22,155 INFO    MainThread:281514 [wandb_init.py:init():831] calling init triggers
2025-06-25 11:04:22,155 INFO    MainThread:281514 [wandb_init.py:init():836] wandb.init called with sweep_config: {}
config: {'batch_size': 32, 'epochs': 50, 'learning_rate': 0.001, 'model': 'DCNN08', 'fault_type': 'outer', 'augmentation': 'fft', 'seed': 258, 'multiplyby': 20, '_wandb': {}}
2025-06-25 11:04:22,155 INFO    MainThread:281514 [wandb_init.py:init():872] starting backend
2025-06-25 11:04:22,358 INFO    MainThread:281514 [wandb_init.py:init():875] sending inform_init request
2025-06-25 11:04:22,360 INFO    MainThread:281514 [wandb_init.py:init():883] backend started and connected
2025-06-25 11:04:22,361 INFO    MainThread:281514 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2025-06-25 11:04:22,361 INFO    MainThread:281514 [wandb_init.py:init():956] updated telemetry
2025-06-25 11:04:22,363 INFO    MainThread:281514 [wandb_init.py:init():980] communicating run to backend with 90.0 second timeout
2025-06-25 11:04:22,895 INFO    MainThread:281514 [wandb_init.py:init():1032] starting run threads in backend
2025-06-25 11:04:22,934 INFO    MainThread:281514 [wandb_run.py:_console_start():2453] atexit reg
2025-06-25 11:04:22,934 INFO    MainThread:281514 [wandb_run.py:_redirect():2301] redirect: wrap_raw
2025-06-25 11:04:22,934 INFO    MainThread:281514 [wandb_run.py:_redirect():2370] Wrapping output streams.
2025-06-25 11:04:22,934 INFO    MainThread:281514 [wandb_run.py:_redirect():2393] Redirects installed.
2025-06-25 11:04:22,935 INFO    MainThread:281514 [wandb_init.py:init():1078] run started, returning control to user process
2025-06-25 11:04:22,936 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:04:22,938 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:04:22,939 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:04:34,364 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:04:34,368 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:04:34,368 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:04:34,371 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:04:34,373 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:04:34,373 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:04:34,374 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:04:34,376 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:04:34,376 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:04:34,376 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:04:34,379 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:04:34,379 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:04:34,380 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:04:34,382 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:04:34,382 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:04:34,382 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:04:34,385 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:04:34,385 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:04:34,494 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:04:34,498 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:04:34,498 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:04:34,499 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:04:34,501 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:04:34,501 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:04:34,502 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:04:34,505 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:04:34,505 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:04:34,805 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:04:34,808 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:04:34,808 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:14,720 WARNING MsgRouterThr:281514 [router.py:message_loop():63] [no run ID] message_loop has been closed
Traceback (most recent call last):
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 27, in _read_message
    return self._sock_client.read_server_response(timeout=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 235, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 220, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router.py", line 56, in message_loop
    msg = self._read_message()
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 29, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-06-25 11:06:14,722 INFO    MsgRouterThr:281514 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
2025-06-25 11:06:14,944 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:26,195 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:26,195 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:26,262 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:26,352 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:26,352 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:26,372 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:26,939 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:26,939 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:26,959 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:26,980 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:26,980 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:26,998 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:27,019 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:27,019 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:27,038 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:27,353 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:27,353 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:27,374 INFO    MainThread:281514 [wandb_run.py:_finish():2219] finishing run tth-bitt-universidade-federal-de-santa-catarina/ottawa2023-fault-detection/u1yysjyp
2025-06-25 11:06:27,374 ERROR   MainThread:281514 [wandb_init.py:init():1619] [no run ID] error in wandb.init()
Traceback (most recent call last):
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1609, in init
    return wi.init(run_settings, run_config, run_printer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 852, in init
    self._wl.finish_all_active_runs()
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_setup.py", line 163, in finish_all_active_runs
    run.finish()
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 391, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 436, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2207, in finish
    return self._finish(exit_code)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 391, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2220, in _finish
    with telemetry.context(run=self) as tel:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/lib/telemetry.py", line 42, in __exit__
    self._run._telemetry_callback(self._obj)
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 759, in _telemetry_callback
    self._telemetry_flush()
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 772, in _telemetry_flush
    self._backend.interface._publish_telemetry(self._telemetry_obj)
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/interface/interface_shared.py", line 60, in _publish_telemetry
    self._publish(rec)
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/interface/interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 174, in send_record_publish
    self.send_server_request(server_req)
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 154, in send_server_request
    self._send_message(msg)
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 151, in _send_message
    self._sendall_with_error_handle(header + data)
  File "/home/thiago-bittencourt/fc-projeto-final/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
           ^^^^^^^^^^^^^^^^^^^^^
BrokenPipeError: [Errno 32] Broken pipe
2025-06-25 11:06:27,580 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:28,170 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:28,170 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:28,192 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:28,346 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:28,347 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:28,366 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:28,387 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:28,387 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:28,406 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:29,673 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:29,673 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:29,692 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:29,713 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:29,713 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:29,732 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:29,753 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:29,754 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:29,790 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:29,811 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:29,811 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:29,830 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:29,852 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:29,852 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:29,871 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:30,381 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:30,382 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:30,462 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
2025-06-25 11:06:30,887 INFO    MainThread:281514 [jupyter.py:save_ipynb():404] [no run ID] not saving jupyter notebook
2025-06-25 11:06:30,887 INFO    MainThread:281514 [wandb_init.py:_pre_run_cell_hook():581] [no run ID] pausing backend
2025-06-25 11:06:33,398 INFO    MainThread:281514 [wandb_init.py:_post_run_cell_hook():592] [no run ID] resuming backend
